### PR TITLE
[NPS-559] Adds glightbox from npm, adds scripts to setup it for nps

### PR DESCRIPTION
**Pull request by mrhyper**
Adds glightbox via npm
Adds scripts that setup glightbox files for shopware scss and js compilers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,24 @@
...
+```
+
+Install npm dependencies
+```$xslt
+npm install
+```
+
+Run script that setups GLightbox
+```$xslt
+npm run glightbox-setup
```
Comment (jacekwilczynski) [inline README.md:19]: Could we perhaps put this, together with `npm install`, in some custom npm script? Or use npm hooks to run this after installation?

File: package.json [hunk 2]
```diff
@@ -6,6 +6,11 @@
   "author": "notebooksbilliger.de AG",
   "license": "proprietary",
   "dependencies": {
+    "cpy-cli": "^3.1.0",
     "glightbox": "^2.0.5"
+  },
+  "devDependencies": {},
+  "scripts": {
+    "glightbox-setup": "cpy './node_modules/glightbox/dist/js/glightbox.js' './src/Resources/app/storefront/src/js/'; cpy './node_modules/glightbox/dist/css/glightbox.css' './src/Resources/app/storefront/src/scss/vendors' --rename='glightbox.scss'"
```
Comment (jacekwilczynski) [inline package.json]: Why use an npm package instead of bash's `cp`?

File: src/Resources/app/storefront/src/main.js [hunk 3]
```diff
@@ -1 +1,14 @@
...
+// const PluginManager = window.PluginManager;
+// PluginManager.register('MainTheme', MainTheme);
+// // PluginManager.register('GLightbox', GLightbox);
+//
+// // Necessary for the webpack hot module reloading server
+// if (module.hot) {
+//     module.hot.accept();
+// }
+//
+// console.log('main.js');
```
Comment (jacekwilczynski) [inline src/Resources/app/storefront/src/main.js]: Uncomment or delete :-)

**Comment by grovas** [comment]
You should also raise plugin version in `composer.json` and add information about changes in `CHANGELOG.md`.

**Review by jacekwilczynski — APPROVED** [review]

**Pull request merged**